src/main/groovy/com/avast/gradle/dockercompose/ServiceInfo.groovy

```diff
@@ -1,20 +1,10 @@
 package com.avast.gradle.dockercompose
 
-import groovy.transform.Immutable
-
-@Immutable
 class ServiceInfo {
     String name
-    ServiceHost serviceHost
-    /* Mapping from exposed to forwarded port. */
-    Map<Integer, Integer> tcpPorts
-    String containerHostname
-    /* Docker inspection */
-    Map<String, Object> inspection
-    String getContainerId() { inspection.Id }
-
-    String getHost() { serviceHost.host }

```

Comment:
For sake of backward compatibility, what about keeping these methods so they would proxy to the first instance? Like `String getHost() { serviceInstanceInfos[0].getHost() }`

But maybe nobody uses these methods so keep it as is.